lib/entry/StateManager.js

```diff
@@ -131,7 +131,11 @@ module.exports = function(liveEntry) {
     }
 
     function updateStreamInfo(currentTime) {
-        if ((currentTime - lastStreamInfoUpdateTime) >= streamInfoUpdateInterval) {
+        let timeDiff = Math.abs(currentTime.getTime() - lastStreamInfoUpdateTime.getTime());
+
+        if (+timeDiff >= +streamInfoUpdateInterval) {
+            // todo: important to decide whether to remove or leave call to getEntryServerNodeId, since it is called directly from liveEntry
+            // can the server node id change during live session after short stream stop start (without stopping session)?
             return getEntryServerNodeId()
```

Comment:
since you got rid of the function getEntryServerNodeId this is either dead code or a potential fatal.